templates/travis-ci/.travis.yml.tmpl

```diff
@@ -1,15 +1,36 @@
+{{- $travisCI := . }}
 # Auto-generated by fogg. Do not edit
 # Make improvements in fogg, so that everyone can benefit.
 dist: bionic
 language: python
 sudo: required
 python: 3.6
+env:
+  global:
+    - BUILD_START=$(date +%s)
+    - BUILDEVENTS_BUILD_ID=${TRAVIS_BUILD_ID}
 
 install:
+  {{- if .Buildevents }}
+  - STEP_START=$(date +%s)
+  - STEP_NAME=install.${TRAVIS_JOB_NUMBER}
+  - STEP_SPAN_ID=$(echo ${STEP_NAME} | sum | cut -f 1 -d \ )
+  {{- end }}
   - pip install awscli --upgrade
   - aws --version
   - FOGG_VERSION="v{{ .FoggVersion }}" travis_retry make setup
+  {{- if .Buildevents }}
+  - mkdir bin # this is in theory supposed to exist already
+  - curl -L -o $HOME/bin/buildevents https://github.com/honeycombio/buildevents/releases/latest/download/buildevents-linux-amd64
```

Comment:
Should we consider pinning the version? I know they have at least once broken the tool. Maybe just worry about that when it happens?